### PR TITLE
Remove useless ELF_GETSHDRSTRNDX test

### PR DESCRIPTION
**Pull request by Rtoax**

### CODE REVIEW

**Review by danobi — COMMENTED** [review]
Wrong file? Maybe you meant cmake/FindLibElf.cmake?

Also I wonder why we're even testing for `elf_getshdrstrndx`. Doesn't look like any parts of the build use it. Maybe some other cmake module transitively requires it...

**Comment by fbs** [comment]
I think the whole file is copied from bcc

**Comment by viktormalik** [comment]
I don't think that we use it. It'd be better to remove the test completely.

**Comment by Rtoax** [comment]
> I don't think that we use it. It'd be better to remove the test completely.

OK

File: cmake/FindLibElf.cmake [hunk 1]
```diff
@@ -43,15 +43,4 @@ FIND_PACKAGE_HANDLE_STANDARD_ARGS(LibElf "Please install the libelf development
...
-CHECK_CXX_SOURCE_COMPILES("#include <libelf.h>
-int main() {
-  Elf *e = (Elf*)0;
-  size_t sz;
-  elf_getshdrstrndx(e, &sz);
-  return 0;
-}" ELF_GETSHDRSTRNDX)
-SET(CMAKE_REQUIRED_LIBRARIES)
-
 mark_as_advanced(LIBELF_INCLUDE_DIRS LIBELF_LIBRARIES ELF_GETSHDRSTRNDX)
```
Comment (viktormalik) [inline cmake/FindLibElf.cmake:46]: Should be removed here, too.

**Review by danobi — APPROVED** [review]

**Pull request merged**